lib/formatters/csv-bal.cjs

```diff
@@ -61,7 +62,28 @@ function buildBalIDs(numero, version) {
   }
 }
 
-function buildRow(voie, numero, position, {includesAlt = false, version = '1.3'}) {
+function addIncludeAlt(voie, numero, row) {
+  if (voie.nomVoieAlt) {
+    Object.keys(voie.nomVoieAlt).forEach(o => {
+      row['voie_nom_' + o] = voie.nomVoieAlt[o]
+    })
+  }
+
+  if (numero.lieuDitComplementNomAlt) {
+    Object.keys(numero.lieuDitComplementNomAlt).forEach(o => {
+      row['lieudit_complement_nom_' + o] = numero.lieuDitComplementNomAlt[o]
+    })
+  }
+}
+
+function addIncludeAllLang(voie, numero, row) {
```

Comment:
Même souci que pour `addIncludeAlt` :
_La fonction `addIncludeAllLang` produit des effets de bord qui apportent des mutations sur la valeur de `row` sans avoir de valeur en retour.
Idéalement, il serait plus rigoureux qu'elle renvoie une nouvelle valeur pour `row`._

_(Sinon, à défaut, placer cette fonction dans le corps de la fonction `buildRow`)_

Reply:
Si je met ca dans la fonction buildRow, le lint crache `Function buildRow has a complexity of 23. Maximum allowed is 20` du coup j'ai mis un retour